bin/config/config-yargs.js

```diff
@@ -52,14 +52,9 @@ const getSchemaInfo = (path, property, subProperty) => {
 	return findPropertyInSchema(current, property, subProperty);
 };
 
-const CONFIG_GROUP = "Config options:";
-const BASIC_GROUP = "Basic options:";
-const MODULE_GROUP = "Module options:";
-const OUTPUT_GROUP = "Output options:";
-const ADVANCED_GROUP = "Advanced options:";
-const RESOLVE_GROUP = "Resolving options:";
-const OPTIMIZE_GROUP = "Optimizing options:";
-const DISPLAY_GROUP = "Stats options:";
+const { GROUPS } = require("../constants");
+
+const { CONFIG_GROUP, BASIC_GROUP, MODULE_GROUP, OUTPUT_GROUP, ADVANCED_GROUP, RESOLVE_GROUP, OPTIMIZE_GROUP, DISPLAY_GROUP } = GROUPS;
```

Comment:
Split it into lines instead on one single line IMO.

Reply:
✨